.ci/pytorch/build.sh

```diff
@@ -82,6 +82,12 @@ if ! which conda; then
   fi
 else
   export CMAKE_PREFIX_PATH=/opt/conda
+
+  # Set Additional library and include path for py3.12
+  if [ "$ANACONDA_PYTHON_VERSION" = "3.12" ]; then
```

Comment:
Ho this one also only need PATH for 3,12?

Reply:
I believe this is not needed here. Let me remove it

Reply:
Yes. Its needed here. I removed another place from conda install

Reply:
So my question still stands why is this only for 3.12? Are we just not picking up these libs for other versions?

Reply:
This is workaround for https://github.com/pytorch/pytorch/issues/119557 . Maybe issue is in generated docker. Image, let me look into this further